### PR TITLE
[13.0] [MIG] base country state translatable

### PR DESCRIPTION
**Pull request by poonlap**
@StephanRozendaal @pedrobaeza 
I re-submit the PR for base_country_state_translatable module for 13.0.

### CODE REVIEW

File: base_country_state_translatable/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 initOS GmbH
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline base_country_state_translatable/__manifest__.py]: Make license lines https:

Comment (poonlap) [reply base_country_state_translatable/__manifest__.py]: Done.

File: base_country_state_translatable/models/res_country.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2017 initOS GmbH
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline base_country_state_translatable/models/res_country.py:2]: Please do all license lines.

**Comment by leemannd** [comment]
Supeseed https://github.com/OCA/partner-contact/pull/1050

**Pull request closed without merging**